template/hooks/common/post-code-deploy/post-code-deploy.sh

```diff
@@ -17,11 +17,8 @@ deployed_tag="$4"
 repo_url="$5"
 repo_type="$6"
 
-# 1 if is ACSF, 0 if not.
-[[ ! -f "/mnt/files/$AH_SITE_GROUP.$AH_SITE_ENVIRONMENT/files-private/sites.json" ]]
```

Comment:
Additionally, I think it's safer / more portable to use `$site.$target_env` rather than `$AH_SITE_GROUP.$AH_SITE_ENVIRONMENT`, but not sure how much you want to take on in this PR...